interface/src/app/greenelab/stats.spec.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Unit tests.
+ */
+
+// Test the hyperGeometricTest function
+describe('Test the hypergeometric test function in stats.js', function() {
+  beforeEach(module('greenelab.stats'));
+
+  var MathFuncts;
+
+  beforeEach(function() {
+    inject(function($injector) {
```

Comment:
Although this `beforeEach()` block works, it adds some unnecessary complexity. It can be simplified to either:
```
beforeEach(inject(function($injector) {
  MathFuncts = $injector.get('MathFuncts');
});
```
or:
```
beforeEach(inject(function(_MathFuncts_) {
  MathFuncts = _MathFuncts_;
});
```